python/py-pgspecial/Portfile

```diff
@@ -13,15 +13,14 @@ description         Meta-commands handler for Postgres Database
 long_description    This package provides an API to execute meta-commands \
                     (AKA “special”, or “backslash commands”) on PostgreSQL.
 
-python.versions     27 34 35 36
+python.versions     27 34 35 36 37
 
 homepage            https://pypi.python.org/pypi/${python.rootname}/
 master_sites        pypi:[string index ${python.rootname} 0]/${python.rootname}
 distname            ${python.rootname}-${version}
 
-checksums           md5     6e617cb1915f414d6ecdc075d77d5f87 \
-                    rmd160  ba666f257b7f54c1bb2be05b91e9c41f10fc1c5b \
-                    sha256  eadb0108cdbcf8b38a69bcc9e403b352dbd6d30622e417f48e659180150ee1b6
+checksums           rmd160  35f6e53c5faa0a3d8da4d18ef5e96216f3ebf258 \
+                    sha256  60e81ed02c3a2d43a4eb70d5bbdeea7b0c5244580d3017edf89a6fd66e6503fb
```

Comment:
It is now customary to put in a "size" field in addition to the rmd160 and sha256.